stylesheets/utils/utilities.css

```diff
@@ -45,3 +45,72 @@
 .p-sticky {
   position: sticky;
 }
+
+/* Text */
+
+.text-left {
+  text-align: left;
+}
+
+.text-right {
+  text-align: right;
+}
+
+.text-center {
+  text-align: center;
+}
+
+.gray-text {
+  color: var(--shadow-color);
+}
+
+.text-sm {
+  font-size: 0.8rem;
+}
+
+.text-md {
+  font-size: 1rem;
+}
+
+.text-lg {
+  font-size: 1.5rem;
+}
+
+.text-xl {
+  font-size: 2rem;
+}
+
+.text-xxl {
+  font-size: 2.5rem;
+}
+
+/* Heading */
+h1 {
+  font-size: 2.5rem;
+  font-weight: 500;
```

Comment:
You can create a more better visual hierarchy by chaging font-weight. Start from 700 to h1 and h2, and decrease by 100 for next set of two and again by 100 to other set.

Reply:
Thanks, that helps. Made the changes.